### DISCUSSION TITLE
19.07.2023 Area A Data Modeling

### DISCUSSION THREAD

**Discussion by hampusnasstrom** [FAIRmat-NFDI/AreaA-data_modeling_and_schemas#55] [Data Modeling Meeting]
## Agenda
We will discuss:
- [Merge request 1385](https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-FAIR/-/merge_requests/1385)
- `Collection`, `Experiment`, and `Analysis` sections
- Topology

## Meeting Notes
- 28th we discuss graph db
- Topology: Deal in visualizer with not having in any strutctural info
	- Merge 1385
	- I create issue
	- Add Ensemble + Substance into topology
	- Lauri looks how the visualizer
- Action buttons: Mainly the API endpoint
- Experiment and Collection we add ActivityReference and EntityReference with name
	- I will test template with Experiment activities

## Tasks

**Comment by andreaa93** [comment]
I feel we miss a SubstanceReference

**Comment by andreaa93** [comment]
or is there any other way you planned to do that?

**Comment by andreaa93** [comment]
I would discuss with you a normalize function for Experiment:

every time that save button is hit, all the activities present are put automatically in an ExperimentStep list depending on their timestamp. This allows to easily create Activities and then to have them grafted in the overarching Experiment automatically. It would also solve a severe problem I already mentioned to @hampusnasstrom concerning the Activities parsed from tabular parser, i. e., the parser is not able to append many different subclasses of Activities all together in the ExperimentStep
